File: class05/python/AP0511-CalculadoraBasica.py
# CalculadoraBasica — Aula 05
# Calculadora com as 4 operações básicas

a = float(input("Digite o primeiro número: "))
b = float(input("Digite o segundo número: "))
print(f"Soma: {a + b}")
print(f"Subtração: {a - b}")
print(f"Multiplicação: {a * b}")
if b != 0:
    print(f"Divisão: {a / b:.2f}")
else:
    print("Divisão: Não é possível dividir por zero")

File: class05/java/AP0511-CalculadoraBasica.java
import java.util.Scanner;
public class CalculadoraBasica {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Primeiro número: ");
        double a = sc.nextDouble();
        System.out.print("Segundo número: ");
        double b = sc.nextDouble();
        System.out.println("Soma: " + (a+b));
        System.out.println("Subtração: " + (a-b));
        System.out.println("Multiplicação: " + (a*b));
        if (b != 0) System.out.printf("Divisão: %.2f%n", a/b);
        else System.out.println("Divisão: Não é possível dividir por zero");
        sc.close();
    }
}